packages/devtools_app/lib/src/screens/memory/panes/leaks/LEAK_TRACKING.md

```diff
@@ -116,6 +116,34 @@ like [the example app does](https://github.com/polina-c/spikes/blob/master/leaki
 
 You can pass information, that will help you troubleshoot the leak, to the optional parameter `details`.
 
+TODO(polina-c): explain how to choose which classes to instrument.
+
+### Detect leaks in your Dart app
+
+For Dart appliocations: 
+
+1. Reference [memory_leak_tracker](https://github.com/polina-c/spikes/blob/master/memory_leak_tracker/README.md):
+
+TODO(polina-c): productize the library `memory_leak_tracker`.
+
+```
+  memory_leak_tracker:
+    git:
+      url: git://github.com/polina-c/spikes.git
+      path: memory_leak_tracker
+```
+
+2. [Instrument](#instrument) your classes.
+
+3. Run your app with additional flags: `dart --observe:8181 main.dart`, find the line `The Dart VM service is listening on` in console and copy URL like `http://127.0.0.1:8181/etNluJDHZwE=/`.
+
+4. Start [DevTools](https://github.com/flutter/devtools) at master: `flutter run -d macos` and connet to the app with the copyed URL. 
```

Comment:
typo: connect

You may also want to add a link to our getting started guide for users that have not ran DevTools locally before